spec/api-shell-spec.js

```diff
@@ -79,4 +79,48 @@ describe('shell module', () => {
       assert.deepEqual(shell.readShortcutLink(tmpShortcut), change)
     })
   })
+
+  describe('shell.moveItemToTrash', () => {
+    const filePath = path.join(os.tmpdir(), `${Date.now()}.md`)
+    fs.writeFileSync(filePath, '# Hello')
```

Comment:
It should be done before each test is executed, not when the tests list is constructed.
```js
describe('shell.moveItemToTrashAsync()', () = {
  describe('if a file exists', () => {
    beforeEach(() => {
      // TODO: Create a temporary file.
    })
    afterEach(() => {
      // TODO: Make sure the temporary file is removed.
    })
    it('removes the file', () => {
       // TODO: Call the API you're testing.
    })
    it('gets fulfilled', () => {
       // TODO: Call the API you're testing.
    })
    it('gets fulfilled with "undefined"', () => {
       // TODO: Call the API you're testing.
    })
  })

  describe('if a file doesn\'t exist', () => {
    it('gets rejected', () => {
       // TODO: Call the API you're testing.
    })
  })
})
```

Reply:
I partially agree, but I don't think calling assertions in before/after hooks is a good idea